pyfar/classes/coordinates.py

```diff
@@ -98,50 +95,266 @@ def __init__(self, points_1=None, points_2=None, points_3=None,
+            positive x direction. Used for spherical and cylindrical coordinate
+            systems.
+        colatitude : ndarray, double
+            angle in radiant with respect to polar axis (z-axe). Used for
+            spherical coordinate systems.
+        radius : ndarray, double
+            distance to origin for each point. Used for spherical coordinate
+            systems.
+        weights: array like, number, optional
+            weighting factors for coordinate points. The `shape` of the array
+            must match the `shape` of the individual coordinate arrays.
+            The default is ``None``.
+        comment : str, optional
+            comment about the stored coordinate points. The default is
+            ``""``, which initializes an empty string.
         """
+        x, y, z = sph2cart(azimuth, colatitude, radius)
+        return cls(x, y, z, weights=weights, comment=comment)
+
+    @classmethod
+    def from_spherical_side(
+            cls, lateral, polar, radius, weights: np.array = None,
+            comment: str = ""):
+        """Create a Coordinates class object from a set of points in the
+        spherical coordinate system. See
+        :py:mod:`coordinates concepts <pyfar._concepts.coordinates>` for
+        more information.
+
+        Parameters
+        ----------
+        lateral : ndarray, double
+            angle in radiant with respect to horizontal plane (x-y-axe).
+            Used for spherical coordinate systems.
+        polar : ndarray, double
+            angle in radiant of rotation from the x-z-plane facing towards
+            positive x direction. Used for spherical coordinate systems.
+        radius : ndarray, double
+            distance to origin for each point. Used for spherical coordinate
+            systems.
+        weights: array like, number, optional
+            weighting factors for coordinate points. The `shape` of the array
+            must match the `shape` of the individual coordinate arrays.
+            The default is ``None``.
+        comment : str, optional
+            comment about the stored coordinate points. The default is
+            ``""``, which initializes an empty string.
+        """
+        x, z, y = sph2cart(polar, np.pi / 2 - lateral, radius)
+        return cls(x, y, z, weights=weights, comment=comment)
+
+    @classmethod
+    def from_spherical_front(
+            cls, phi, theta, radius, weights: np.array = None,
+            comment: str = ""):
+        """Create a Coordinates class object from a set of points in the
+        spherical coordinate system. See
+        :py:mod:`coordinates concepts <pyfar._concepts.coordinates>` for
+        more information.
+
+        Parameters
+        ----------
+        phi : ndarray, double
+            Tangle in radiant of rotation from the y-z-plane facing towards
+            positive y direction. Used for spherical coordinate systems.
```

Comment:
How about calling this the frontal angle instead of phi. I'm still not very happy because theta and phi are very arbitrary and the definition we use does not comply with most of the literature using spherical coordinate systems.

Reply:
I took it from an AES standard 56 for loudspeaker measurements but have no strong preference. If we change it, it would be best if we find a better name for theta as well